tests/test_md.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+import os
+
+from metadata import get_sound_class
+
+# NOTE: currently running this out of stem_mixer/script, might need to adjust paths later on
+
+@pytest.mark.parametrize(
```

Comment:
maybe we add an example that does not belong to musdb/brid but still has a clear sound class?

Reply:
ok i can add this